docs/configuration.md

```diff
@@ -413,7 +413,7 @@ template {
   # command will only run if the resulting template changes. The command must
   # return within 30s (configurable), and it must have a successful exit code.
   # Consul Template is not a replacement for a process monitor or init system.
-  # Please see the [Command](#command) section below for more.
+  # Please see the Commands section in the README for more.
```

Comment:
I removed the link because the links don't render in the markdown code blocks, it just shows the raw markdown. 

And yeah, a relative URL is probably better than the absolute URL one I used in mode.md. I'll update that.